CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - added: Add disableSurveyModal option
 - changed: Disable Home scene swap and FIO cards if configured
 - changed: Allow some chains to be disabled in env.json
+- changed: Use quoteAllocations to create amount rows in StakeModifyScene
```

Comment:
I think the difference (using quoteAllocations instead of stakeAssets and rewardAssets to generate rows) is enough to consider it a "changed" but there is also a small visual difference where the scene might render additional rows after the first change quote (previously, it was know how many rows to show upon entering the scene)